### ISSUE TITLE
Tab completion

### ISSUE THREAD

**Issue by jtbates** [torch/DEPRECEATED-torch7-distro#77]
Tab completion is not working correctly for me on OS X.  Other users in our lab report the same problem.  I have built Torch with readline.  I'm using OS 10.7.4 (Lion).  I don't have problems with tab completion in other programs such as ipython and I don't have tab completion problems with torch on Linux.  I have not tested other versions of OS X.

Sometimes I get no output from tab completion and sometimes I get weird output like this

```
t7> nn.A<tab><tab>
nn.Aebug.   nn.Ao       nn.Aofile(  nn.Aok.
```

Whereas on Linux, I get this:

```
t7> nn.A<tab><tab>
nn.Abs.           nn.AbsCriterion.  nn.Add.  
```

**Comment by jtbates** [comment]
Issue initially reported by @atcold in the torch-tutorials repo.

**Comment by jtbates** [comment]
Answered in the newsgroup here: https://groups.google.com/forum/?fromgroups=#!topic/torch7/uGCTI9Mi5nQ
TLDR: brew install readline; brew link readline
